fem/bilininteg_vecmass.cpp

```diff
@@ -98,7 +98,7 @@ void VectorMassIntegrator::AssemblePA(const FiniteElementSpace &fes)
 template<const int T_D1D = 0,
          const int T_Q1D = 0>
 static void PAVectorMassApply2D(const int NE,
-                                const Array<double> &_B,
+                                const Array<double> &B_,
```

Comment:
I assume this was hitting a system call. This is why I try to avoid underscore prefixed variables. I think this fix is fine, but it's probably worth thinking about reworking this naming strategy.

Reply:
I think this was hitting a macro called `_B` from one of the standard headers that cygwin uses and was resulting in an error. We've run into this before but somehow these names starting with `_` sneaked in again. :-)